### PR TITLE
Entity

### PR DESCRIPTION
**Pull request by CraigThorburn**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -8,6 +8,8 @@ services:
       - ./src/:/src
       - ./data/:/src/data
     command: ["bash", "run.sh"]
+    environment:
+      - "COMPOSE_CONVERT_WINDOWS_PATHS=1"
```
Comment (EntilZha) [inline docker-compose.yml]: not on windows, I assume that this fixes some problem so +1

File: environment.yaml [hunk 2]
```diff
@@ -1,5 +1,6 @@
-name: base
+name: qa
 dependencies:
+  - python=3
```
Comment (EntilZha) [inline environment.yaml]: Should be 3.6, good catch though

File: environment.yaml [hunk 3]
```diff
@@ -9,5 +10,6 @@ dependencies:
   - requests
   - click
   - flask
+  - tqdm
```
Comment (EntilZha) [inline environment.yaml]: Surprised this isn't included already, but good to include if it isn't

File: environment.yaml [hunk 4]
```diff
@@ -1,5 +1,6 @@
-name: base
+name: qa
```
Comment (EntilZha) [inline environment.yaml]: Should probably keep the name unchanged for now. I had issues today with getting a separate gpu image to build on base and had to change, did you have similar trouble? I could see the default docker installing everything in a separate environment and assuming `run.sh` and `cli` run `source activate qa`

**Comment by CraigThorburn** [comment]
It was my mistake, I accidentally submitted a pull request for this repo rather than our project's repo.  The changes for windows definitely helped us (2 team members are using Windows Subsytem for Linux)

**Pull request closed without merging**